scheduling/job_scheduling.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+'''
+Author : Mohit Kumar
+Job Sequencing Problem implemented in python
+'''
+
+
+class Scheduling:
+
+    def __init__(self, jobs):
+        self.jobs = jobs
+
+    def schedule(self, n, deadline):
+        self.sdl = deadline
+        self.J = []
+        self.J.append(self.jobs[1])
+        self.x = 2
+        while self.x < n:
+            self.K = self.J.copy()
+            self.K.append(self.jobs[self.x])
+            self.x = self.x + 1
+            self.feasibility = self.feasible(self.K, self.sdl)
+            if self.feasibility is True:
+                self.J = self.K.copy()
+
+        return self.J
+
+    def feasible(self, K, fdl):
+        self.tmp = K
+        self.isFeasible = True
+
+        self.i = 0
+        self.j = 1
+        self.k = 0
```

Comment:
```suggestion
        i = 0
        j = 1
        k = 0
```
Do not bind these variables to the object.